include/libdnf/rpm/package_query.hpp

```diff
@@ -587,6 +587,16 @@ class PackageQuery : public PackageSet {
     // TODO(dmach): consider removing the installed packages during the filtering
     void filter_priority();
 
+    /// Keep in query only packages that are installed but not available in any enabled
+    /// repository. Also installed packages that are only part of non-active modules
+    /// are considered extras.
```

Comment:
Nice improvement, What about to `considered extras` change  to `considered as extras`?

Reply:
Done. Sorry for all those mistakes in my English.